core/dictgen/src/rootcling_impl.cxx

```diff
@@ -4537,6 +4537,7 @@ int RootClingMain(int argc,
    }
 
    std::ostream &dictStream = (!gOptIgnoreExistingDict && !gOptDictionaryFileName.empty()) ? fileout : std::cout;
+   bool isACLiC = linkdef.length() > 16 && !linkdef.compare(linkdef.size() - 16, 16, "_ACLiC_linkdef.h");
```

Comment:
Why not just using `strstr`?

Reply:
Because it's slower (scans the whole string) and less precise (picks up any occurrence, not just at the end)?

Reply:
But is also clearer and consistent to other cases of implementing `isACLiC` in the codebase.